src/Symfony/Component/JsonPath/Test/JsonPathAssertionsTrait.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\JsonPath\Test;
+
+use Symfony\Component\JsonPath\JsonCrawler;
+use Symfony\Component\JsonPath\JsonPath;
+
+/**
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ *
+ * @experimental
+ */
+trait JsonPathAssertionsTrait
```

Comment:
I suggest adding `@psalm-requires-extend \PHPUnit\Framework\Assert` as this trait relies on the static methods of PHPUnit and is broken if using it in a different way. Both psalm and phpstan support this annotation and will flag invalid usages of the trait.